src/mmw/js/src/modeling/views.js

```diff
@@ -880,7 +884,10 @@ var ScenarioToolbarView = Marionette.CompositeView.extend({
     },
 
     templateHelpers: function() {
-        var gisData = this.currentConditions.getGisData().model_input,
+        // We don't need to apply modifications to gis_data here
+        // because it's are only downloadable from this view if
```

Comment:
> because it's are only downloadable from this view if

Is there a word missing?

Reply:
An extra word. Two for the price of one. "because it's ~are~ only downloadable from this view if"